CHANGELOG.md

```diff
@@ -13,7 +13,8 @@
 * Appended headers to configuration files generated by `teleport-update`. [#56578](https://github.com/gravitational/teleport/pull/56578)
 
 Enterprise:
-* Updated AWS ID plugin to honor Role and Access Request logs.
+* Updated AWS Identity Center plugin to honor Role and Access Request locks.
+* Updated AWS Identity Center plugin to not provision users when Teleport is not acting as a SAML IdP for AWS
```

Comment:
nit: missing period at end of line (the automation puts it there so you don't have to, but if you skip the automation...)